components/organisms/ContributorCard/contributor-card.tsx

```diff
@@ -33,14 +35,20 @@ const ContributorCard = ({ className, contributor, topic, repositories, range }:
   const [showPRs, setShowPRs] = useState(false);
   const { chart, repoList, meta } = useContributorPullRequestsChart(profile.githubName, topic, repositories, range);
   const languageList = useContributorLanguages(profile.githubName);
+  const { data: user } = useFetchUser(profile.githubName, {
+    revalidateOnFocus: false,
+  });
+
+  const { is_maintainer } = user ?? {};
 
   return (
     <Card className={className && className}>
       <div className="flex flex-col gap-3">
-        <div className="flex w-full justify-between items-center gap-2">
+        <div className="flex items-center justify-between w-full gap-2">
           <CardProfile {...profile} totalPRs={meta.itemCount} />
-          <div>
+          <div className="flex flex-col items-end gap-2">
             <CardHorizontalBarChart withDescription={false} languageList={languageList} />
+            {is_maintainer && <Badge text="maintainer" />}
```

Comment:
nit: consistency

```suggestion
            {!!is_maintainer && <Badge text="maintainer" />}
```

Reply:
Do we need the double negation `!!` if `isMaintainer` is already a boolean type? I've only seen this when values need to be forced into their "truthy" state in javascript

Reply:
Looks like we do this elsewhere

https://github.com/open-sauced/insights/blob/807430da7a0eba23402e85159cfc8fc1ff719dc0/components/atoms/InsightBadge/insight-badge.tsx#L8

👍🏼 

more just curious now why we have this convention

Reply:
One of the reasons why this conversion is added is because the value is coming from an API call, and will be undefined initially until the fetch is complete, hence we are converting undefined to `boolean`. If we don't, we get a typescript error.

Reply:
![](https://media.giphy.com/media/3og0IMJcSI8p6hYQXS/giphy.gif)